### DISCUSSION TITLE
WhatsApp status

### DISCUSSION THREAD

**Discussion by avydigital** [devlikeapro/waha#65] [Ideas]
Hi,
I don't see in your documents the possibility to upload a status to WhatsApp.
It would be cool if you could enable this in the next version

**Comment by allburov** [comment]
Hi!
Sorry, the feature is not available right now in WhatsApp Web (even one that you can use via browser). 
Because the engines we use work with WhatsApp Web it's not possible to add the feature :( 

If we're talking about statuses [contacts see for 24 hours](https://faq.whatsapp.com/643144237275579/?helpref=uf_share)
